datavault-webapp/src/main/webapp/WEB-INF/freemarker/vaults/security.ftl

```diff
@@ -104,19 +110,17 @@
                         </div>
                     </div>
 
-                    <#assign securityExpr = "hasPermission('${vault.ID}', 'vault', 'MANAGE_ROLES')">
-                    <@sec.authorize access=securityExpr>
-                        <div class="form-group ui-widget col-sm-10 form-confirm">
-                            <div class="checkbox">
-                                <input class="form-check-input" id="confirm-checkbox" type="checkbox"
-                                       name="assigningRole" checked="checked"/>
-                            </div>
-                            <label for="confirm-checkbox" class="control-label">Assign role to previous data
-                                owner?</label>
+                    <div class="form-group ui-widget col-sm-10 form-confirm">
+                        <div class="checkbox">
+                            <input class="form-check-input" id="confirm-checkbox" type="checkbox"
+                                   name="assigningRole" checked="checked"/>
                         </div>
+                        <label for="confirm-checkbox" class="control-label">Assign role to previous data
+                            owner?</label>
+                    </div>
                     </@sec.authorize>
 
-                    <@sec.authorize access="hasRole('ROLE_IS_ADMIN')">
+                    <@sec.authorize access="hasRole('ROLE_IS_ADMIN') or hasPermission('${vault.groupID}', 'GROUP', 'CAN_ORPHAN_SCHOOL_VAULTS')">
```

Comment:
Does this work? I was getting a syntax error on the embedded `${vault.groupID}`, but may have been arising from something else.

Reply:
Yeah, this works fine for me, but if it's looking like it's causing problems on your machine we should get to the bottom of what it is

Reply:
I haven't tested with this PR specifically, it was when I originally updated that template I attempted something similar. 

Though, maybe I just did it wrong. It's probably fine.